src/App.css

```diff
@@ -1,38 +1,22 @@
-.App {
-  text-align: center;
+.navbar-brand {
```

Comment:
This can be exactly replaced with the Bootstrap classes "d-flex mt-0 me-2".

Reply:
Looks like lines 1-3 are actually just unnecessary. Adding "d-flex mt-0 me-2" in place of that class caused the buttons to become extra close together and caused an oddity next to the profile button. 
With new change: ![image](https://github.com/isthisdotusernametaken/WeSchedule/assets/42187569/60ac8e0e-f96e-4349-b2c2-a41f42553ed3)
Removing unnecessary class altogether: ![image](https://github.com/isthisdotusernametaken/WeSchedule/assets/42187569/4f80cf1b-19b9-44a3-a567-2fce2e9011e4)

The change you suggested looks cleaner.

Where did you get the Bootstrap class from?

Reply:
Ah, to use Bootstrap classes, we don't have a CSS style rule at all, just add "d-flex mt-0 me-2" to the className list in the JSX.